Fix-Website/Stylesheets/toc.css

```diff
@@ -1,17 +1,25 @@
 .TOC {
-    background-color: #FAFAFA;
-    height: 800px;
-    width: 400px;
+    background-color: #fafafa;
     float:left;
+    height: 800px;
     margin-top: 10px;
-    position: static;
+    overflow-y: auto;
+    position: fixed;
+    width: 400px;
 }
 .Info {
+    margin-left: 420px;
     position: static;
-    margin-left: 20px;
-    float:left;
+    width: 1200px;
```

Comment:
We should generally never specify width for items in our CSS. As that is much better handled by the CSS framework we are using.

The problem with specifying width here (e.g. `margin-left`) is that other things are dependent upon the exact width. Moreover, the website doesn't react well when the browser window is resized.

We should keep this as-is for now, since we know we will be beating the website with a CSS stick later. But for reference, read this page: http://getbootstrap.com/css/. Bootstrapp is a proto-typical CSS framework that abstracts out the need for you to specify exact widths of things.